sites/waterworld/server/api/fragments/global-thought-leaders/index.js

```diff
@@ -0,0 +1,9 @@
+const company = require('./company');
```

Comment:
You should be able to combine all these thought leader fragments into on with ... on calls